swarm/__init__.py

```diff
@@ -0,0 +1,11 @@
+#  -*- coding: utf-8 -*-
+
+from swarm import activations
+
+
+def get_activation(name):
+    try:
+        activfunc = getattr(activations, name)
+    except AttributeError as ex:
+        raise AttributeError(f"Unable to find activation {name} in torch.nn") from ex
+    return activfunc
```

Comment:
This is smart, lets us just look up and put in any function name that's in the torch lib library. Good idea. Better than my approach.